packages/legend-application-query/src/components/__tests__/QueryEditor.dataspace.test.tsx

```diff
@@ -106,3 +106,70 @@ test(
     // await waitFor(() => getByText(dataspaceViewModal, 'this is template with inline query'));
   },
 );
+
+test(
```

Comment:
[NEXT PR] we need to increase test coverage here.
1. can we test when no light graph is returned in the result. 
2. can we maybe add to the engine roundtrip and test the analysis dataspace endpoint and do unit testing on some of the build graph/ build query builder state you have build.
3. can we ensure some of the CRUD methods on dataspace queries are still working and saving as expcted.